db/drone_dao.py

```diff
@@ -124,6 +135,61 @@ def can_self_configure(drone: discord.Member) -> bool:
     return can_self_configure_drone is not None and bool(can_self_configure_drone['can_self_configure'])
 
 
+def is_battery_powered(drone: discord.Member) -> bool:
+    battery_powered_drone = fetchone('SELECT is_battery_powered FROM drone WHERE id = :discord', {'discord': drone.id})
+    return battery_powered_drone is not None and bool(battery_powered_drone['is_battery_powered'])
+
+
+def deincrement_battery_minutes_remaining(member: discord.Member = None, drone_id: str = None):
+    drone_record = None
```

Comment:
Is this used for anything? `drone_record` gets reassigned in both branches immediately